notebooks/financial-docs-10k-example.ipynb

```diff
@@ -0,0 +1,160 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "0",
+   "metadata": {},
+   "source": [
+    "##### In this example, we will use request library to make API call to Aryn Partitioning Service. We'll send a financial document (a 10k) to the service and then extract a table from the JSON object returned. We'll turn that table into a pandas dataframe and then perform some analysis on it.\n",
+    "\n",
+    "##### To access the Aryn Partitioning Service for fast, GPU-powered performance go to [aryn.ai/sign-up ](aryn.ai/sign-up) and sign up to get a free API key for the service. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Run pip install aryn-sdk first\n",
+    "import aryn_sdk\n",
+    "import boto3\n",
+    "import pandas as pd\n",
+    "import os\n",
+    "from io import BytesIO\n",
+    "from aryn_sdk.partition import partition_file, tables_to_pandas"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# visit https://www.aryn.ai/cloud to get a key\n",
+    "aryn_api_key = os.environ['ARYN_API_KEY'] "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "file_name = '3m_10k.pdf'\n",
+    "s3 = boto3.client('s3')\n",
+    "s3.download_file(\"aryn-public\", \"partitioner-blog-data/3m_10k.pdf\", file_name)\n",
+    "f = open(file_name, 'rb')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "partitioned_file = partition_file(f, aryn_api_key, extract_table_structure=True, use_ocr=True, selected_pages=[23])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "partitioned_file"
```

Comment:
Could be combined with the earlier cell

Reply:
Want a cell to highlight the JSON that's outputted.